pkg/service/performance_policy_service.go

```diff
@@ -60,7 +60,7 @@ func onlyPerformancePolicy(objs []*model.PerformancePolicy) (*model.PerformanceP
 	return objs[0], nil
 }
 
- 
+
 // GetPerformancePolicysByID - method returns associative a array of pointers of type "PerformancePolicy", filter by Id
 func (svc *PerformancePolicyService) GetPerformancePolicysByID(pool *model.Pool, fields []string) (map[string]*model.PerformancePolicy, error) {
```

Comment:
Why is this returning a map of id to PerformancePolicy?  It should return a single PerformancePolicy.